scribejava-apis/src/main/java/com/github/scribejava/apis/DoximityApi20.java

```diff
@@ -0,0 +1,67 @@
+package com.github.scribejava.apis;
+
+import com.github.scribejava.apis.service.Doximity20ServiceImpl;
+import com.github.scribejava.core.builder.api.DefaultApi20;
+import com.github.scribejava.core.model.OAuthConfig;
+import com.github.scribejava.core.model.OAuthConstants;
+import com.github.scribejava.core.model.ParameterList;
+import com.github.scribejava.core.oauth.OAuth20Service;
+
+import java.util.Map;
+
+public class DoximityApi20 extends DefaultApi20 {
+
+    protected DoximityApi20(){
+    }
+
+    private static class InstanceHolder {
+        private static final DoximityApi20 INSTANCE = new DoximityApi20();
+    }
+
+    public static final DoximityApi20 instance(){
+        return InstanceHolder.INSTANCE;
+    }
+
+    private static final String AUTHORIZE_URL = "https://auth.doximity.com/oauth/authorize?";
```

Comment:
https://travis-ci.org/scribejava/scribejava/jobs/159339878#L1670

[ERROR] src/main/java/com/github/scribejava/apis/DoximityApi20.java:[25,5] (coding) DeclarationOrder: Static variable definition in wrong order.